src/api/goods.ts

```diff
@@ -1,11 +1,27 @@
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll(): Promise<Good[]> {
+export const getAll = (): Promise<Good[]> => {
```

Comment:
If you use arrow functions - it's better to type variable rather than function parts.
Create separate type for getAll, get5First and getRed
`export const getAll: CallbackType = () => {...}`